### ISSUE TITLE
Rename master to nightly

### ISSUE THREAD

**Issue by LucioFranco** [amethyst/website-legacy#119]
This would change the reference to master to nightly on the website. This is mostly for the url and docs page.

Edit: Updated latest to master

**Comment by dotellie** [comment]
While I do agree with this change in spirit, I don't think calling it nightly without actually having releases is appropriate :sweat_smile:

**Comment by LucioFranco** [comment]
@magnonellie I agree but I don't know how we can facilitate that, without creating a bunch of tags 😢 .

Maybe, we can have a nightly branch? That gets merged when the nightly docs get built?

**Comment by LucioFranco** [comment]
I totally freaked on this, I meant master -> nightly. Don't know what I was thinking. I want to make this change because the docs are actually built nightly so it would make more sense.
